signl4/README.md

```diff
@@ -0,0 +1,88 @@
+## Overview
+
+Use the [SIGNL4][1] integration to send Datadog alerts to SIGNL4 teams and seamlessly take actions on these alerts within the SIGNL4 app.
+
+Connect SIGNL4 to Datadog in order to:
+- Trigger and resolve incidents from Datadog
+- Tackle incidents and set up escalation policies as they occur
+- Set up a daily reminder of who is on-call
+
+![SIGNL4 App][2]
+
+## Setup
+
+### Follow these steps in SIGNL4:
+
+1. If not already done you can create your SIGNL4 account at [signl4.com][1].
+
+2. In your SIGNL4 app find your SIGNL4 webhook address including your team secret under Settings -> APIs.
+
+### Follow these steps in Datadog:
+
+1. Open the **Integrations** page from the sidebar.
+
+2. Use the search bar to search for "webhooks".
+
+3. Once the **Webhooks** tile appears, hover and click on "Install".
+
+4. Scrol down and then click **New** to create a new webhook.
+
+5. Under the section **Name** and **URL**, enter a meaningful name and paste the **SIGNL4 Webhook URL** including your team secret. The URL looks like follows:
+
+```
+https://connect.signl4.com/webhook/<team-secret>?ExtIDParam=alertId&ExtStatusParam=alertTransition&ResolvedStatus=Recovered
+```
```

Comment:
Indent 4 spaces

Reply:
The code block needs to be indented 4 spaces so the numbered list won't break on the docs site.